Dockerfile

```diff
@@ -14,8 +14,9 @@ RUN python -m pip install .
 # Runtime Image
 FROM python:3.11-slim-bullseye as runtime
 COPY --from=build /opt/venv /opt/venv
+COPY --from=build /app/log_conf.json /app/log_conf.json
```

Comment:
this isn't still in `/app` is it?  I know I've flipped it around a few times.

Reply:
It's in `/app/` in the build image from `WORKDIR /app` and `COPY log_conf.json .`

Reply:
Aha!  gotcha... yep I see it 🤦‍♂️.

Maybe the better fix is to not include it in the build image and instead just use it in the runtime image?

Reply:
FYI I put this change on `main` cause the container is failing in cloud run now, still think we should not put it in the build image!

Reply:
Made the change to just put it in the runtime image